src/Kunstmaan/GeneratorBundle/Generator/KunstmaanGenerator.php

```diff
@@ -255,7 +261,7 @@ protected function installDefaultPageTemplates($bundle)
         $contents = file_get_contents($dirPath . 'view.html.twig');
         if (strpos($contents, '{% extends ') === false) {
             GeneratorUtils::prepend(
-                "{% extends '" . $bundle->getName() . ":Layout:layout.html.twig' %}\n",
+                "{% extends '" . $bundle->getName() . "/Layout/layout.html.twig' %}\n",
```

Comment:
There should a symfony 4 check here as `App/Layout/layout.html.twig` doesn't work.

SF4:
```
"{% extends 'Layout/layout.html.twig' %}\n",
```

SF3.4
```
"{% extends '" . $bundle->getName() . ":Layout:layout.html.twig' %}\n",
```